modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -79,6 +79,16 @@ You can create your Anypoint VPC inside any of the available CloudHub regions. +
 The recommended region to use might vary depending on how you connect to your Anypoint VPC. If you are using a VPN tunnel, you might want to choose the CloudHub region closest to your data center. +
 However, if you are peering with your private AWS VPC, you need to create your Anypoint VPC in the same AWS region.
 
+== FAQ: How to deploy or move applications to a VPC
+
+MuleSoft applications are deployed to an environment (dev, sandbox, and so on). 
+
+When the deployment region matches the VPC region:
+
+* If the environment is already associated with a VPC at the time of application deployment, the app resides inside the VPC.
+* If the environment is associated with a VPC after the app deployment, the app will not reside inside the VPC until it is restarted.
+* When the deployment region does not match the VPC region, the app is deployed outside of the VPC. This means it will not have access to any connectivity methods configured for the VPC. 
```

Comment:
"This means that the app does not have access to...."

Can it be moved later? The FAQ title implies moving as well as deploying.